components/compositing/compositor.rs

```diff
@@ -1483,7 +1483,7 @@ impl<Window: WindowMethods> IOCompositor<Window> {
             _ => initialize_png(&*self.gl, width, height)
         };
 
-        profile(ProfilerCategory::Compositing, None, self.time_profiler_chan.clone(), || {
+        profile(ProfilerCategory::Compositing, None, &self.time_profiler_chan.clone(), || {
```

Comment:
Hey @jdm, and thanks for the review ! Could you point me to any hint which would allow me to pass &self.time_profiler_chan to the closure without cloning it first ? The compiler throws some borrow issues at me : 

` error[E0500]: closure requires unique access to `self` but `self.time_profiler_chan` is already borrowed
    --> /home/ignition/Documents/Projects/rust/servo/components/compositing/compositor.rs:1486:80
     |
1486 |         profile(ProfilerCategory::Compositing, None, &self.time_profiler_chan, || {
     |                                                       -----------------------  ^^ closure construction occurs here
     |                                                       |
     |                                                       borrow occurs here
...
1490 |             let size = webrender_traits::DeviceUintSize::from_untyped(&self.frame_size.to_untyped());
     |                                                                        ---- borrow occurs due to use of `self` in closure
1491 |             self.webrender.render(size);
1492 |         });
     |          - borrow ends here

error: aborting due to previous error(s)
 `